lib/view_model/settings/settings_view_model.dart

```diff
@@ -264,55 +69,31 @@ abstract class SettingsViewModelBase with Store {
 
   @computed
   TransactionPriority get transactionPriority {
-    final priority = _settingsStore.priority[_walletType];
+    final priority = _settingsStore.priority[walletType];
 
     if (priority == null) {
-      throw Exception('Unexpected type ${_walletType.toString()}');
+      throw Exception('Unexpected type ${walletType.toString()}');
     }
 
     return priority;
   }
 
-  @computed
-  BalanceDisplayMode get balanceDisplayMode =>
-      _settingsStore.balanceDisplayMode;
-
-  @computed
-  bool get shouldSaveRecipientAddress =>
-      _settingsStore.shouldSaveRecipientAddress;
-
-  @computed
-  bool get allowBiometricalAuthentication =>
-      _settingsStore.allowBiometricalAuthentication;
-
   @computed
   ThemeBase get theme => _settingsStore.currentTheme;
 
   bool get isBitcoinBuyEnabled => _settingsStore.isBitcoinBuyEnabled;
 
   final Map<String, String> itemHeaders;
-  List<List<SettingsListItem>> sections;
   final SettingsStore _settingsStore;
   final YatStore _yatStore;
-  final WalletType _walletType;
-  final BiometricAuth _biometricAuth;
-
-  @action
-  void setBalanceDisplayMode(BalanceDisplayMode value) =>
-      _settingsStore.balanceDisplayMode = value;
+  final WalletType walletType;
+  final  WalletBase<Balance, TransactionHistoryBase<TransactionInfo>,
+              TransactionInfo> _wallet;
 
   @action
   void setFiatCurrency(FiatCurrency value) =>
       _settingsStore.fiatCurrency = value;
 
-  @action
-  void setShouldSaveRecipientAddress(bool value) =>
-      _settingsStore.shouldSaveRecipientAddress = value;
-
-  @action
-  void setAllowBiometricalAuthentication(bool value) =>
-      _settingsStore.allowBiometricalAuthentication = value;
-
   @action
   void toggleTransactionsDisplay() =>
       actionlistDisplayMode.contains(ActionListDisplayMode.transactions)
```

Comment:
I can't see any use for those, please double-check on them and if so, please remove them and delete this class since its not used anymore